### ISSUE TITLE
change the label for URLs

### ISSUE THREAD

**Issue by troughc** [culturecreates/footlight-app#277]
Use URL instead of https/http. 

update the label for the input field to display URL instead of https:// (for fields that take URLs).

**Comment by saumier** [comment]
@troughc I think I meant to assign this to October release as it is low priority. Please confirm. I have moved it to October but if I am in error let me know. Thx.

**Comment by troughc** [comment]
will revisit the UX with Stephanie before starting work

**Comment by troughc** [comment]
Tested: PASSED
